index.html

```diff
@@ -926,10 +914,50 @@ <h2>The <code>application/vc+jwt</code> Media Type</h2>
   </table>
 </section>
 
-<section id="conformance"></section>
+<section id="vc-ld-jwt-media-type">
+  <h2><code>application/vc+ld+jwt</code></h2>
+  <p>
+    This specification registers the <code>application/vc+ld+jwt</code> Media Type specifically for
+    identifying a <a data-cite="rfc7519#section-3">JWT</a> conforming to the Verifiable Credentials JWT format in the `typ` header.
+  </p>
+  <table>
+    <tr>
+      <td>Type name: </td>
+      <td>application</td>
+    </tr>
+    <tr>
+      <td>Subtype name: </td>
+      <td>application/vc+ld+jwt</td>
+    </tr>
+    <tr>
+      <td>Required parameters: </td>
+      <td>None</td>
+    </tr>
+    <tr>
+      <td>Encoding considerations: </td>
+      <td>
+        <code>application/vc+ld+jwt</code> values are encoded
+        as a series of base64url encoded values (some of which may be the
+        empty string) each separated from the next by a single period
+        ('.') character.
```

Comment:
this sounds exactly as IANA registry for JWTs, I think we need to add text how it's different from that (ie +jwt and vc parts)

Reply:
Right, but perhaps it is better to address that in a smaller PR.

Reply:
I filed https://github.com/w3c/vc-jwt/issues/71